lang/cs/Org.Apache.REEF.Tang/Implementations/ClassHierarchy/AvroClassHierarchy.cs

```diff
@@ -206,6 +236,33 @@ public INode GetNode(string fullName)
             return ret;
         }
 
+
+        public INode GetNode(string fullName, string aliasLanguage)
```

Comment:
Lot of IClassHierarchy methods are used in Tang internally only. Tang users mostly should only use Configuration, Configuration module and Injector methods. I would mark this new method as Obsolete for now. We will need a separate Jira to mark those interface as obsolete and move them to internal.